INSTALL-from-source.md

```diff
@@ -32,26 +37,27 @@ Here is a `yum` command Redhat/CentOS:
 
 To build documentation you need:
 
-* texinfo
+* [`texinfo`](https://www.gnu.org/software/texinfo/) for building the TeXInfo docs
 
 Add that to the `apt-get` or `yum` command above.
 
 # Simplified approach
 
-
+```console
    $ $SHELL ./autogen.sh
    $ make && make check
-
+```
 
 This performs the step below steps up to but not including
 "Building".
 
 # Creating and running configure script
 
 
+```console
 	$ autoreconf -f -i
 	$ ./configure
```

Comment:
If you write "This performs.." then maybe use the same command and arguments as in autogen.sh?

And in "This performs the step below steps...", the first "step" should be removed.

Reply:
Ok. I think this is done now in fccab2fc Please double check.